### PR TITLE
Grab docker log using a soft link in local-up-cluster

### PR DESCRIPTION
**Pull request by dims**
**What this PR does / why we need it**:
Would be useful to debug problems like timeouts and missing images etc
for the local e2e jobs.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -253,6 +253,9 @@ if [[ ${CONTAINER_RUNTIME} == "docker" ]]; then
     CGROUP_DRIVER=$(docker info | grep "Cgroup Driver:" | cut -f3- -d' ')
     echo "Kubelet cgroup driver defaulted to use: ${CGROUP_DRIVER}"
   fi
+  if [[ -f /var/log/docker.log && ! -f ${LOG_DIR}/docker.log ]]; then
```
Comment (fejta) [inline hack/local-up-cluster.sh]: Are we certain $LOG_DIR exists?

**Review by fejta — COMMENTED** [review]
/lgtm
/hold

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (dims) [reply hack/local-up-cluster.sh]: following the pattern like the rest of the file. if LOG_DIR is not set we default to `/tmp` and if it is set we assume it's created already

**Pull request merged**